### PR TITLE
PAM-33911: Multi-tenancy in streaming analytics

### PR DESCRIPTION
**Pull request by kailol-sag**
This needs to be cherry-picked into the branch for 10.15.

### CODE REVIEW

**Review by skom-sag — APPROVED** [review]

File: content/apama/analytics-introduction-bundle/deploying-apps.md [hunk 1]
```diff
@@ -43,7 +43,10 @@ This only applies if you are using Apama 10.3 or later.
...
 
-Apama can only be used in a single-tenant microservice. Therefore, the microservice manifest must set the isolation level to PER_TENANT. The reason for this is that Apama's Cumulocity IoT transport connectivity plug-in can only communicate with the tenant to which it is deployed. Therefore, having an Apama shared between multiple tenants is invalid.
+Apama can be used in either a single-tenant microservice or a multi-tenant microservice. 
+Therefore, the microservice manifest must set the isolation level to either PER_TENANT or MULTI_TENANT.
+When Apama is used in a multi-tenant microservice, the Apama application must be written to be multi-tenant aware. 
+For more information, see [Working with multi-tenant deployments]({{< link-apama-webhelp >}}index.html#page/pam-webhelp%2Fco-ConApaAppToExtCom_cumulocity_working_with_multi_tenant_deployments.html) in the Apama documentation.
 
 The following permissions are required by the microservice in order to start up and and use all features in the Cumulocity IoT transport from EPL. These are set with requiredRoles in the microservice manifest.
```
Comment (BeateRixen) [inline content/apama/analytics-introduction-bundle/deploying-apps.md:51]:
```suggestion
The following permissions are required by the microservice in order to start up and and use all features in the {{< product-c8y-iot >}} transport from EPL. These are set with requiredRoles in the microservice manifest.
```

File: content/apama/analytics-introduction-bundle/developing-apps.md [hunk 2]
```diff
@@ -154,6 +154,23 @@ Note that the above description assumes that you are connecting to a tenant wher
 CUMULOCITY_TENANT=my_custom_tenant
 ```
 
+If the project needs to run locally in a multi-tenant environment, enable the multi-tenant support and provide the name of the multi-tenant microservice to use 
+by configuring following properties in the *CumulocityIoT.properties* file under the Cumulocity IoT client:
```
Comment (BeateRixen) [inline content/apama/analytics-introduction-bundle/developing-apps.md:158]:
```suggestion
by configuring the following properties in the *CumulocityIoT.properties* file under the {{< product-c8y-iot >}} client:
```

File: content/apama/analytics-introduction-bundle/deploying-apps.md (hunk 1 above)
Comment (kailol-sag) [reply content/apama/analytics-introduction-bundle/deploying-apps.md]: ok, done. And also removed the duplicate "and".

**Review by skom-sag — APPROVED** [review]

File: content/apama/analytics-introduction-bundle/developing-apps.md (hunk 2 above)
Comment (kailol-sag) [reply content/apama/analytics-introduction-bundle/developing-apps.md]: ok, done

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**